core/src/main/java/io/opencensus/stats/View.java

```diff
@@ -42,13 +42,13 @@
   public abstract Measure getMeasure();
 
   /**
-   * Dimensions (a.k.a Tag Keys) to match with the associated {@link Measure}.
-   * If no dimensions are specified, then all stats are recorded. Dimensions must be unique.
+   * Columns (a.k.a Tag Keys) to match with the associated {@link Measure}.
+   * If no dimensions are specified, then all stats are recorded. Columns must be unique.
```

Comment:
Sure, I'll add this to Javadoc in the View PR https://github.com/census-instrumentation/opencensus-java/pull/462.